.github/workflows/ci-ts.yaml

```
@@ -0,0 +1,52 @@
+        uses: actions/setup-node@v4
+        with:
+          node-version: '22'
+          cache: 'pnpm'
+          
+      - name: Install dependencies
+        run: pnpm install
+                        
+      - name: Run tests with Docker
+        run: pnpm test
+        env:
+          DISPLAY: :99
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)